### DISCUSSION TITLE
Accounts do not preserve sort

### DISCUSSION THREAD

**Discussion by death2cupbots** [firefly-iii/firefly-iii#4311] [Firefly III]
**Bug description**
I am running Firefly III version 5.4.6, and my problem is:
When I try to rearrange transactions from the same date on an account page and then navigate away from the page, if I go back the transactions are back in their original order.

I'm reasonably certain I've had success with this before.

**Steps to reproduce**

1. Have or create multiple transactions with the same date and (at the very least) the same source or destination account.
2. Drag the transactions so that they are in a different order and notice that they flash green after you have dragged them.
3. Refresh the page (or leave the page and come back)
4. Notice that the transactions have returned to their original sort (by date and then by transaction number).

**Extra info**

Debug information generated at 2021-01-15 10:37:38 America/New_York for Firefly III version **5.4.6**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.4.6 |
| Firefly III API | 1.4.0 |
| PHP | 7.4.13 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 33fe0fbe-cfee-496b-81fa-a926233790d2 |
| Using docker? | false |
| Telemetry | false |
| Layout |  |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | fpm-fcgi |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql |
| Current driver | mysql |
| DB version | 15 (exp. 15) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2021-01-01 00:00:00 |
| Session end | 2021-01-31 00:00:00 |
| Session first | 2019-06-17 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36 |

**Bonus points**

- [x] I searched and nobody reported this bug before
- [ ] I have added a stack trace from my log files 
- [ ] I have added a screenshot.
- [x] I was able to replicate it on the demo site https://demo.firefly-iii.org/

**Comment by JC5** [comment]
Indeed. I can't recall but I believe the function has been in/out of commission between versions. I'll pick it up as a bug, thanks for reminding me.
